app/api_alpha/tests/test_ads.py

```diff
@@ -1013,7 +1019,12 @@ def clean_users_in_response(d):
                 "username": username,
                 "organization_name": organization_name,
                 "email": email,
-            }
+            },
+            {
+                "username": "johndoe",
+                "organization_name": organization_name,
+                "email": "",  # Cet utilisateur est créé sans email dans le setup de cette classe de test
+            },
```

Comment:
```suggestion
                "email": "",  # This user is already created in the setUp() function without an email. If the user already exists it is returned without being updated. 
            },
```

Je suis un peu pénible 😬 tous nos commentaires sont en anglais dans le code, on va essayer de garder la convention.